packages/terra-framework-docs/src/terra-dev-site/doc/table/Examples/DefaultTable.1.doc.mdx

```diff
@@ -1,4 +1,10 @@
 import { Badge } from "terra-table/package.json?dev-site-package";
 import DefaultTable from "./DefaultTable?dev-site-example";
 
+# Default Table
```

Comment:
Do we want to update this terminology to 'Basic Table' ?

Reply:
I think either term would be clear enough for consumers to understand.